api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -88,4 +89,8 @@ message UdpProxyConfig {
   // The default if not specified is false, that means each data chunk is forwarded
   // to upstream host selected on first chunk receival for that "session" (identified by source IP/port and local IP/port).
   bool use_per_packet_load_balancing = 7;
+
+  // Configuration for :ref:`access logs <arch_overview_access_logs>`
+  // emitted by the udp_proxy.
```

Comment:
You should document the new dynamic metadata formatters here or in the main filter docs.